src/odemis/util/spectrum.py

```diff
@@ -75,6 +75,35 @@ def get_wavelength_per_pixel(da):
     raise KeyError("No MD_WL_* metadata available")
 
 
+def get_time_per_pixel(da):
```

Comment:
In the "spectrum" module, it feels a little out of place. In addition, it's so simple that probably the caller could do it itself. But anyway, I understand the parallel with get_wavelength_per_pixel(), so let's keep it there for now.